README.md

```diff
@@ -217,6 +217,19 @@ calls return unexpected 401 or 403 errors.
     $client->setSubject($user_to_impersonate);
     ```
 
+#### How to use a specific JSON key
+
+If you want to a specific JSON key instead of using `GOOGLE_APPLICATION_CREDENTIALS` environment variable, you can do this:
+
+```
```

Comment:
set the syntax for php here:

```
```php
```

Reply:
@bshaffer Fixed and git squashed my commit. Thanks.